From f3607b7f4bcd9955f140e15d8857637c84f2b604 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Wed, 25 Jul 2018 11:49:45 +0300
Subject: [PATCH 403/448] powerpc/configs/sdk_dpaa: enable the Cortina PHY driver

Cortina PHYs are present on T4240RDB and T2080RDB. Enable the driver
by default.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
---
 arch/powerpc/configs/sdk_dpaa.config |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/configs/sdk_dpaa.config b/arch/powerpc/configs/sdk_dpaa.config
index 5cdd8f4..78db185 100644
--- a/arch/powerpc/configs/sdk_dpaa.config
+++ b/arch/powerpc/configs/sdk_dpaa.config
@@ -4,3 +4,4 @@ CONFIG_FSL_SDK_QMAN=y
 CONFIG_FSL_SDK_DPA=y
 CONFIG_FSL_SDK_FMAN=y
 CONFIG_FSL_SDK_DPAA_ETH=y
+CONFIG_CORTINA_PHY=y
-- 
1.7.1

